Mathlib/AlgebraicGeometry/EllipticCurve/VariableChange.lean

```diff
@@ -170,21 +170,16 @@ instance : MulAction (VariableChange R) (WeierstrassCurve R) where
           - C.r * C.t * C.u⁻¹ ^ 6 * ↑C'.u⁻¹ * (C'.s * 2 + W.a₁) * pow_mul_pow_eq_one 5 C'.u.inv_mul
           + C.u⁻¹ ^ 6 * (C.r ^ 3 - C.t ^ 2) * pow_mul_pow_eq_one 6 C'.u.inv_mul
 
-@[simp]
```

Comment:
Which problem did you solve by removing the `simp` attribute from these lemmas?

Reply:
I discussed this with @Multramate privately. The problem is that there might be customly defined change of variables, and it didn't seem useful that `simp` would just destruct every change of variable. It would be better, in my opinion, for each change of variables to come with simp lemmas.

Reply:
You could collect them in a "simp set"; then one needs to remember (and write down) only one name.